src/modules/code.js

```diff
@@ -0,0 +1,140 @@
+import { list, todo } from './variables.js';
+
+let todoList = [];
```

Comment:
- [ ] Kindly be noticed that the array should have some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\].

Reply:
ok